app/src/main/AndroidManifest.xml

```diff
@@ -61,25 +61,22 @@
         <activity
             android:name=".activity.LogicalAnalyzerActivity"
             android:configChanges="screenSize|orientation"
-            android:screenOrientation="landscape"/>
+            android:screenOrientation="landscape" />
         <activity android:name=".activity.SensorDataLoggerActivity" />
-
-        <activity android:name=".activity.MultimeterActivity"
-            android:screenOrientation="portrait"/>
-
         <activity
-            android:name=".activity.PowerSourceActivity"
-            android:windowSoftInputMode="stateHidden|adjustResize"
+            android:name=".activity.MultimeterActivity"
             android:screenOrientation="portrait" />
-
-        <activity android:name=".activity.WaveGeneratorActivity"
-            android:screenOrientation="userLandscape"/>
-
+        <activity
+            android:name=".activity.PowerSourceActivity"
```

Comment:
Why there are changes to these instruments?

Reply:
This is due to repositioning of code as I added the setting activity tag below main activity